### ISSUE TITLE
解説書1.3.5の意図 第1段落について

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#1582]
A11Y Slackでの指摘

> > 特に認知障害のある人として、フォームへの入力を容易にする
>
>これもやっぱり違和感があるので、「特に認知障害のある人に対して、フォームへの入力を容易にする」みたいなほうがいいんじゃないかと。

原文：
> The intent of this Success Criterion is to ensure that the purpose of a form input collecting information about the user can be programmatically determined, so that user agents can extract and present this purpose to users using different modalities. The ability to programmatically declare the specific kind of data expected in a particular field makes filling out forms easier, especially for people with cognitive disabilities.

https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html

現行の訳文：
>この達成基準の意図は、ユーザエージェントが異なるモダリティを使用し、入力フォームの目的を抽出して利用者に提示できるように、ユーザエージェントが利用者に関する情報を収集するフォーム入力の目的を、プログラムによる解釈が可能にすることである。特定の分野で期待される特定の種類のデータをプログラムによって宣言する機能は、特に認知障害のある人として、フォームへの入力を容易にする。

1文目は、so that構文がうまく取れていない？
2文目はオリジナルの指摘とは別に、in a particular field を「特定の分野」としているが、入力ファームの話なので、「フィールド」とすべきなのでは？

以下はどうか：
1文目について
（so thatを結果と取る）
この達成基準の意図は、利用者に関する情報を収集するフォームの入力の目的を、プログラムによる解釈を可能にして、その結果、ユーザエージェントが異なるモダリティを使用し、その目的を抽出して利用者に提示できるようにすることである。
（so thatを目的と取る）
この達成基準の意図は、ユーザエージェントが異なるモダリティを使用して、利用者にフォームの入力の目的を抽出して提示するために、利用者に関する情報を収集するその入力の目的を、プログラムによる解釈を可能にすることである。

2文目：
特定のフィールドで期待される特定の種類のデータをプログラムで宣言する機能により、特に認知障害を持つ人々にとって、フォームへの入力が容易になる。

**Comment by bakera** [comment]
（so thatを結果と取る）のほうを採用したいです。
